automation/test.sh

```diff
@@ -372,27 +372,33 @@ if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} ]]; then
     export KUBEVIRT_E2E_FOCUS="\\[sig-storage\\]|\\[storage-req\\]"
     export KUBEVIRT_E2E_SKIP="Migration"
   elif [[ $TARGET =~ vgpu.* ]]; then
-    export KUBEVIRT_E2E_FOCUS=MediatedDevices
+    export KUBEVIRT_E2E_FOCUS="MediatedDevices"
```

Comment:
Not sure whether the same label selector would need to get applied here. @xpivarc might know more than me.

```suggestion
    export KUBEVIRT_E2E_FOCUS="MediatedDevices"
    export ginko_params="$ginko_params --label-filter=\"needs-gpu\""
```

Reply:
@xpivarc Your thoughts?

Reply:
> Not sure whether the same label selector would need to get applied here. @xpivarc might know more than me.

I do not understand the question.
This is a different marking, appearing in `mdev_configuration_allocation_test.go`.